drivers/soundwire/intel.h

```diff
@@ -50,7 +58,7 @@ struct sdw_intel {
 
 #define SDW_INTEL_QUIRK_MASK_BUS_DISABLE      BIT(1)
 
-int intel_master_startup(struct platform_device *pdev);
-int intel_master_process_wakeen_event(struct platform_device *pdev);
+int intel_link_startup(struct intel_link_dev *ldev);
+int intel_link_process_wakeen_event(struct intel_link_dev *ldev);
```

Comment:
maybe struct ancillary_device as parameters in both places to keep consistency with existing code?

Reply:
done